net.lutris.Lutris.yml

```diff
@@ -27,6 +29,15 @@ finish-args:
   - --filesystem=xdg-data/Steam:ro
   - --filesystem=~/.var/app/com.valvesoftware.Steam:ro
   - --env=PATH=/app/bin:/app/runners/bin:/app/utils/bin:/usr/bin:/usr/lib/extensions/vulkan/MangoHud/bin/
```

Comment:
Since the `.Runner` and `.Utility` extension points were removed, it makes sense to remove `/app/runners/bin` and `/app/utils/bin` respectively from `PATH`.

Also you may want to copy `GST_PLUGIN_SYSTEM_PATH` finish-arg from the Wine app.

Reply:
Is there a reason we can safely remove .Utility isnt that needed for mangohud et al.?

Reply:
`.Utility` isn't needed for MangoHud anymore, `com.valvesoftware.Steam.Utility.MangoHud` is EOL.

Reply:
Same with vkBasalt.